content/rs/administering/cluster-operations/updating-certificates.md

```diff
@@ -26,6 +26,26 @@ future sessions.{{% /note %}}
 
 {{% warning %}}The new certificate replaces the equivalent certificate on all nodes in the cluster. Existing certificates are overwritten.{{% /warning %}}
 
+- For Redis Enterprise version 5.4.0 and above use rladmin
```

Comment:
@ushachar Is this correct? Any reason for customers not to update certs with this command?

Reply:
rladmin calls the same API - customers can use either option.